_overviews/jdk-compatibility/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: multipage-overview
+title: Overview
+
+discourse: true
+
+partof: jdk-compatibility
+overview-name: JDK Compatibility
+
+num: 1
+permalink: /overviews/jdk-compatibility/:title.html
+---
+
+Scala runs primarily on the Java Virtual Machine (JVM). As Scala and the JVM improve independently over time, Scala drops compatibility with older versions of the Java Developer Kit (JDK) in order to focus development efforts on supporting new JVM features that benefit Scala.
+
+This table shows the first Scala release in each series that functions on each JDK.
+
+| JDK version | First Scala release supporting JDK version per series |
+|:-----------:|:-----------------------------------------------------|
+| 9           | 2.12.4, 2.11.12                                      |
```

Comment:
Thanks for getting the ball rolling.

The Java 9 row needs an asterisk to mention that support is incomplete. Notably, scalac will not enforce the restrictions of the Java Platform Module System, which means that code that typechecks may incur linkage errors at runtime.

Reply:
Added a full section on it:

https://github.com/scala/docs.scala-lang/pull/1005/files#diff-c509fc8681047cdea35034d7993f8ef2R25

Reply:
2.10.7 includes some minimal changes intended to improve JDK 9 support, in fact, that's the whole reason 2.10.7 even exists

Reply:
Addressed in https://github.com/scala/docs.scala-lang/pull/1005/commits/979f07aa1bcdf3db5faa962c54479254e8e71d3d